### ISSUE TITLE
Add CFSR to variables

### ISSUE THREAD

**Issue by khegewisch** [Google-Drought/WebInterface#40]

**Comment by khegewisch** [comment]
collection is : NOAA/CFSV2/FOR6H
https://ee-api.appspot.com/c56762e50ce569b8dcf81c9314560da7

Justin wants: 

//Maximum_temperature_height_above_ground_6_Hour_Interval
//Minimum_temperature_height_above_ground_6_Hour_Interval
//Latent_heat_net_flux_surface_6_Hour_Average
//Sensible_heat_net_flux_surface_6_Hour_Average
//Precipitation_rate_surface_6_Hour_Average
//Maximum_specific_humidity_at_2m_height_above_ground_6_Hour_Interval
//Minimum_specific_humidity_at_2m_height_above_ground_6_Hour_Interval
//Specific_humidity_height_above_ground
//Downward_Short-Wave_Radiation_Flux_surface_6_Hour_Average
//Downward_Short-Wave_Radiation_Flux_surface_6_Hour_Average
//Volumetric_Soil_Moisture_Content_depth_below_surface_layer
//Potential_Evaporation_Rate_surface_6_Hour_Average



And then compute 

Net Radiation as 
(Downward_Long-Wave_Radp_Flux_surface_6_Hour_Average - Upward_Long-Wave_Radp_Flux_surface_6_Hour_Average) + (Downward_Short-Wave_Radiation_Flux_surface_6_Hour_Average - Upward_Short-Wave_Radiation_Flux_surface_6_Hour_Average)

Horizontal windspeed as
sqrt(u^2 + v^2)

where the u component is the east-west and the v component is north-south
//u-component_of_wind_height_above_ground
//v-component_of_wind_height_above_ground



Eventually we want to compute ASCE Standardized Penman-Monteith potential ET...(who knows what formulation they are using).... but for now using theirs is a good start.

Basically anything we could compute with U of I METDATA we can compute with CFSR.. but CFSR is 6hrly.. so some aggregation will be in order if we want to summarize for folks, or just spit out the 6hrly..

For drought, the values, anomalies, and climos we / most folks will probably be most interested in will be PPT, PET, soil moisture, and latent and sensible heat.  Humidity, temp, net rad, solar, windspeed will be very helpful to identify the drivers of different droughts.  By plotting many anomaly timeseries together, or having easy access to the data for download and then plot together outside of our page will be really useful for looking at different drivers of drought.
